gateway/package.json

```
@@ -9,95 +9,78 @@
-    @apollo/gateway: "2.12.1",
-    @apollo/server: "4.11.3",
-    @opentelemetry/api: "1.9.0",
-    @opentelemetry/sdk-trace-base: "2.0.1",
-    dataloader: "2.2.2",
-    graphql: "16.8.1",
-    graphql-query-complexity: "0.12.0",
-    pino: "8.17.0",
-    redis: "5.8.1"
-=======
-<<<<<<< HEAD
+    "@apollo/gateway": "2.12.1",
```

Biome (2.4.7)

[error] The key `@apollo/gateway` was already declared.

(lint/suspicious/noDuplicateObjectKeys)